### PR TITLE
Run make to build the project

### PR DESCRIPTION
**Pull request by limitedAtonement**
Although anyone running `cmake` probably knows to run `make`, we may as well add it for completeness's sake.

### CODE REVIEW

File: README-sycl.md [hunk 1]
```diff
@@ -258,9 +258,11 @@ mkdir -p build && cd build
 
 # Option 1: Use FP16 for better performance in long-prompt  inference
 cmake .. -DLLAMA_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DLLAMA_SYCL_F16=ON
+make
```
Comment (phymbert) [inline README-sycl.md:261]: We usually only use `cmake`, I would suggest to change to:

```shell
cmake --build . --config Release -j $(nproc)
```

**Review by phymbert — DISMISSED** [review]

**Review by phymbert — COMMENTED** [review]
according to @cebtenzzre comment, forgive my previous comment

**Review by cebtenzzre — APPROVED** [review]

**Comment by NeoZhangJianyu** [comment]
@limitedAtonement 
Thank your fix!

This issue was created by another PR: https://github.com/ggerganov/llama.cpp/pull/6141

The removed code is :
```
# Or, build all binary
cmake --build . --config Release -v
``` 

If good if you provide both make and cmake here.

**Comment by limitedAtonement** [comment]
> If good if you provide both make and cmake here.

I wasn't sure how to provide both, so I gave it the old college try. How does that look?

**Review by phymbert — APPROVED** [review]

**Comment by arthw** [comment]
OK, I will check and update if needed.

Thank you!

**Comment by abhilash1910** [comment]
I think this PR needs another look, from usability point of view .

**Comment by limitedAtonement** [comment]
@abhilash1910 What are your concerns? I used the instructions to build (using `make`, not `cmake --build`).

**Comment by arthw** [comment]
I test and find the parameter "cmake --build .. xxxx" is wrong in common build env.
I fix it by PR: https://github.com/ggerganov/llama.cpp/pull/6755

**Pull request merged**